server/utils/routes.js

```diff
@@ -23,4 +23,10 @@ export const v1Routes = {
   users: {
     show: userRoot,
   },
+  auth: {
+    login: `${v1Root}/login`,
+  },
+  charts: {
+    show: `${v1Root}/data/charts/:chart_id`,
```

Comment:
```suggestion
    show: `${v1Root}/charts/:chart_id`,
```

Reply:
We have a get route that gets the chart form in this format /charts/:chart_id. The data prefix gets the data, maybe charts/data/:chart_id would be better

Reply:
I left another comment above but I think `/charts/:chart_id/data` makes sense.